tests/sys_execve/src/test_sys_execveat.c

```diff
@@ -0,0 +1,376 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why rename the file? It still essentially tests `execv` (`execvat` is just a derivation). Additionally, renaming the file make it much harder to diff with the previous version.

Reply:
Will revert this